gradle/libs.versions.toml

```diff
@@ -0,0 +1,9 @@
+[versions]
+gradle = "8.0.0"
```

Comment:
Maybe `android-gradle-plugin` is a bit more accurate, Gradle version is a different thing. Also, you can use 8.2.0 here as it's the latest version.

Reply:
Agreed 👍